integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
@duglin 

I try to use buildImageWithStdoutStderr, but got a blank stdout.
Could you please give me some hints?

Thanks.